app/application.py

```diff
@@ -90,13 +90,23 @@ def _request_requires_authentication():
            and request.method != 'OPTIONS'
 
 
-@app.errorhandler(MessageSaveException)
-def handle_save_exception(error):
+@app.errorhandler(RasNotifyException)
+def handle_notify_exception(error):
     response = jsonify(error.to_dict())
     response.status_code = error.status_code
     return response
 
 
+@app.errorhandler(MessageSaveException)
+def handle_save_exception(error):
+    return jsonify(status_code=error.status_code)
```

Comment:
Could we just get away with setting the status code instead of returning it in the body?